### PR TITLE
Add CSCS contributors

### PR DESCRIPTION
**Pull request by msimberg**
Do we need orcids for the submission? @aurianer, @biddisco do you already have an id?

### CODE REVIEW

**Comment by hkaiser** [comment]
> Do we need orcids for the submission? @aurianer, @biddisco do you already have an id?

I don't think we needs them. Might be good to have one, however.

**Comment by diehlpk** [comment]
It is optional, but as @hkaiser mentioned, it will not hurt to add one or generate one. Only the submitter needs one which I will be.

**Comment by msimberg** [comment]
> It is optional, but as @hkaiser mentioned, it will not hurt to add one or generate one. Only the submitter needs one which I will be.

All right, thanks. I'll create one for myself at least and add it here. I'll check with @aurianer and @biddisco.

File: paper.md [hunk 1]
```diff
@@ -24,6 +24,15 @@ authors:
   - name: Weile Wei
     orcid: 0000-0002-3065-4959
     affiliation: "1"
+  - name: John Biddiscombe
+    orcid: ???
```
Comment (biddisco) [inline paper.md]: Just registered : https://orcid.org/0000-0002-6552-2833

Comment (msimberg) [reply paper.md]: Thanks! Added.

Comment (aurianer) [reply paper.md]: Same :)
0000-0002-5531-0458

**Pull request merged**